include/fmt/ranges.h

```diff
@@ -415,15 +415,21 @@ struct formatter<
       detail::range_formatter_type<Char, detail::uncvref_type<range_type>>;
   formatter_type underlying_;
   bool custom_specs_ = false;
+  bool use_brackets_ = true;
```

Comment:
I suggest inverting this, i.e. replacing use_brackets_ with no_brackets_ for consistency with the specifier and to make the default false.

Reply:
I'm usually not a fan of double negatives, but it makes sense here.